ecms_base/themes/custom/ecms/ecms.theme

```diff
@@ -201,7 +201,9 @@ function ecms_preprocess_media__file(array &$variables): void {
   $variables['file_title'] = $media->label();
 
   // Grab file filename.
-  $variables['file_filename'] = $media->get('field_file')->entity->getFilename();
+  if (isset($media->get('field_file')->entity)) {
```

Comment:
If there is no file, does any of the remaining code even need to be execuuted? Maybe we can simply return at this point?